machine-learning/setup/00-data-download.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -e
+set -o pipefail
+
+# Set the working directory to the directory of this file
+cd "$(dirname "${BASH_SOURCE[0]}")"
+
+# Use the OpenPBTA bucket as the default.
+URL=${OPENPBTA_URL:-https://s3.amazonaws.com/kf-openaccess-us-east-1-prd-pbta/data}
```

Comment:
Do we need these environment variable conditionals? Not that big a deal, but it seems unlikely we would use any other source without other updates to the script.

Reply:
We probably don't need this for the URL. I can make that change. Maybe next time we cover this module we would have an updated histologies file, though - that's where my head is at.